builder/vmware/iso/builder.go

```diff
@@ -273,6 +274,7 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 			Headless:           b.config.Headless,
 		},
 		&vmwcommon.StepTypeBootCommand{
+			Skip:        b.config.BootCommand == nil,
```

Comment:
this seems redundant. Can we just check if `BootCommand` is nil inside the step?